curriculum/challenges/english/14-responsive-web-design-22/learn-css-animation-by-building-a-ferris-wheel/6169b284950e171d8d0bb16a.md

```diff
@@ -38,7 +38,7 @@ assert(new __helpers.CSSHelp(document).getCSSRules('keyframes')?.[1]?.cssRules?.
 
 ```html
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
This is the last step, it includes a solution at the bottom of the file. Can you add this `lang="en"` to the solution @Liam-boston

Reply:
Yup np